Modules/Segmentation/Voronoi/include/itkVoronoiDiagram2DGenerator.hxx

```diff
@@ -274,8 +274,10 @@ VoronoiDiagram2DGenerator<TCoordinate>::ConstructDiagram()
     buildEdges.push_back(curr);
     EdgeInfo front = curr;
     EdgeInfo back = curr;
-    while (!(rawEdges[i].empty()))
+    auto     maxStop = static_cast<int>(rawEdges[i].size());
```

Comment:
Why is there a static_cast to a signed int here?  What if there are more edges than will fit in the positive range of an integer?

I think maxStop could just be the same type as size() ( likely size_t ).